locales/en/actions.json

```diff
@@ -1,10 +1,13 @@
 {
   "create_move": "Create a move",
+  "create_new_move": "Create another move",
   "previous_day": "Previous day",
   "next_day": "Next day",
   "current_day": "Today",
+  "or": "Or",
   "back": "Back",
   "back_to_dashboard": "$t(actions::back) to dashboard",
+  "return_to_dashboard": "Return to dashboard",
```

Comment:
Maybe we should use the key above so we're not creating another term for the same action?